internal/controlplane/controlplane.go

```diff
@@ -102,6 +109,13 @@ func (d *dashboardClient) Delete(ctx context.Context, obj client.Object) error {
 				return err
 			}
 		}
+
+		ssls, _ := cluster.SSL().List(ctx)
```

Comment:
This operation will completely delete all SSL objects, rather than deleting the objects managed by the current gateway.